Python/setup.py

```diff
@@ -39,7 +39,7 @@ def get_cuda_version(cuda_home):
                 version_str = f.readline().replace('\n', '').replace('\r', '')
                 return version_str.split(" ")[2][:4]
         else:
-            version_str = subprocess.check_output([os.path.join(cuda_home,"bin","nvcc"),"--version"])
+            version_str = subprocess.check_output([os.path.join(cuda_home,"bin","nvcc.exe"),"--version"])

```

Comment:
same, this is being reverted to previous state.  should be without exe

Reply:
Fixed in defa305